src/darwin/Framework/CHIP/MTRDeviceAttestationInfo.h

```diff
@@ -70,15 +70,44 @@ API_AVAILABLE(ios(16.4), macos(13.3), watchos(9.4), tvos(16.4))
  */
 @property (nonatomic, copy, readonly, nullable) NSData * firmwareInfo;
 
+/**
+ * The vendor ID value from the device's Basic Information cluster that was used
+ * for device attestation.  If attestation succeeds, this must match the vendor
```

Comment:
```suggestion
 * for device attestation.  If attestation succeeds, this will match the vendor
```
or
```suggestion
 * for device attestation.  For attestation to succeed, this must match the vendor
```